src/annie.js

```diff
@@ -4,7 +4,7 @@ const customConfig = require(`./config/customConfig.js`)
 const config = require(`./config/global`)
 const commandsLoader = require(`./commands/loader`)
 const Database = require(`./libs/database`)
-const localizer = require(`./libs/localizer`)
+const {localizer, getTargetLocales} = require(`./libs/localizer`)
```

Comment:
Does `localizer` function exist inside /libs/localizer? If I read it correctly there was only getTargetLocales() being exported

Reply:
that is correct, that's a ghost variable, I just pushed a small commit to get rid of it.